README.md

```diff
@@ -37,6 +46,18 @@ logger.Info("Failed to fetch URL.",
 )
 ```
 
+## Development Status: Release Candidate 1
+The current release is `v1.0.0-rc.2`. No further breaking changes are *planned*
```

Comment:
If we add this here, we should also add a check to `make version-check` to ensure this doesn't diverge.